src/client/src/css/jobApplication.css

```diff
@@ -70,24 +70,142 @@
 }
 
 .JAPContainer .textbox {
-    margin: 35px;
+    margin: 15px;
 }
 
 
-@media all and (max-width:800px) {
-    .JAPContainer{
-        width: auto;
+@media all and (max-width:1100px) {
+    
+    .JAPContainer {
+        display: flex;
+        flex-direction: column;
+        width: 800px;
+        height: 600px;
+    }
+
+    .JAPContainer .jobDetails {
+        width: 80%;
+        margin-top: -50px;
+
     }
     
+    .JAPContainer .jobApplicationForm{
+        display: flex;
+        flex-direction: row;
+        margin-top: -30px;
+        margin-left: 85px;
+    }   
     
 }
 
-@media all and (max-width:445px) {
-    .JAPContainer{
-        width: auto;
+
+@media all and (max-width:810px) {
+    
+    .JAPContainer {
+        
+        width: 550px;
+        height: 1000px;
```

Comment:
The footer is sticky, I don't know if that's to be expected.

![image](https://user-images.githubusercontent.com/78051064/230804556-b702303e-1dee-460d-90d2-7d76e2d2740e.png)